src/components/AssetIcon.tsx

```diff
@@ -10,10 +10,11 @@ type AssetIconProps = {
 
 // The icon here is as the placeholder while the icon loads, or if it fails to load.
 
-export const AssetIcon = ({ symbol, ...rest }: AssetIconProps) => {
+export const AssetIcon = ({ symbol, src, ...rest }: AssetIconProps) => {
```

Comment:
We've added `src?: string`, but not updated the types in `AssetIconProps` accordingly.

We might also want to add a comment noting the behavior of `AssetIcon`, i.e. that either `symbol` or `src` should be supplied as arguments, but not both. If both are supplied, we use `src`.

Reply:
@0xApotheosis actually `src?: string` is inherited from `AvatarProps`

Reply:
Perfect, leave as is 👌